injector/src/main/java/edu/ucr/cs/riple/injector/Helper.java

```diff
@@ -395,16 +395,22 @@ public static String getPackageName(String name) {
   }
 
   /**
-   * Corrects Path starting with prefix: {@code file}
+   * Deserializes a Path instance from a string.
    *
-   * @param value Path to file.
+   * @param path Path to file.
    * @return The modified Path.
    */
-  public static String extractPath(String value) {
-    if (!new File(value).exists() && value.startsWith("file:")) {
-      return value.substring("file:".length());
+  public static Path deserializePath(String path) {
+    if (path.startsWith("file://")) {
+      path = path.substring("file://".length());
     }
-    return value;
+    // Keep only one occurrence of "/" from the beginning if more than one exists.
```

Comment:
Does this happen after checking for `file://`?

As I see it, we either get `file://some/path/foo.bar` or `/some/path/foo.bar`. Are we getting _any_ other formats here?

This actually looks like in some places we are serializing filesystem paths and in other's URIs (e.g. somewhere you are doing Path -> String -> [output] and in others Path -> URI -> String -> [output]), so I think there might be a deeper data serialization issue here that this method is somewhat papering over (by parsing the string representations of two different java objects, each with its own parsing and serialization mechanisms to the same thing, which seems more brittle than ensuring we are using either Paths or URIs consistently when we serialize stuff).

Reply:
Yes, direct URIs to strings are coming from [here](https://github.com/uber/NullAway/blob/d6d2978094acec39b50070daae75606c0a8c01a1/nullaway/src/main/java/com/uber/nullaway/fixserialization/location/FieldLocation.java#L48). We should change that in the upcoming version as discussed offline to serialize `realPath`.

Regarding covering all cases, we can either land this PR now to re-enable CI for #111, #113 and then update all path serializations in all annotator code and NullAway to serialize `realPath` in a follow up, or cover all cases within this PR. Please let me know which one you prefer. Thank you